server/lib/config/pbench-server-default.cfg

```diff
@@ -22,7 +22,7 @@ lock-dir = %(lib-dir)s/locks
 [pbench-server]
 user=%(default-user)s
 group=%(default-group)s
-admin-email=%(user)s@localhost
+admin-email=admins@localhost
```

Comment:
Is this a defined alias? Otherwise, isn't email to it going to bounce?

Reply:
All emails will bounce if they are not setup.  In our production environment we use an admin email that resolves outside of the deployment system (which is typical).